backend/docs/auth.md

```diff
@@ -0,0 +1,53 @@
+# <u>Backend Documentation</u>
+</br>
+
+### Auth module
+This module is responsible for all the activities related to authentication.
+
+### API description
+-  [ <b>POST</b> ] : <code> <b> /auth/login </b> </code>
```

Comment:
~[ <b>POST</b> ] : <code> <b> /auth/login </b> </code>~
`/auth/login` [POST]

This looks better what do you say @Kajol-Kumari

Reply:
Yeah. Looks cleaner!